domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/AccessProfileServiceImpl.java

```diff
@@ -19,12 +19,16 @@
 import static uk.gov.hmcts.ccd.definition.store.repository.model.UserRoleModelMapper.toEntity;
 import static uk.gov.hmcts.ccd.definition.store.repository.model.UserRoleModelMapper.toModel;
 
+/**
+ * RDM-10539 - The UserRoles will not be changed in this class as it is only used in the UserRole controller which will
```

Comment:
Could you add this to Jira ticket description / comment please.

Reply:
👍